caper/cromwell_rest_api.py

```diff
@@ -263,17 +255,17 @@ def __request_get(self, endpoint, params=None):
                 url, auth=self._auth, params=params,
                 headers={'accept': 'application/json'})
         except Exception as e:
-            # traceback.print_exc()
-            print(e)
-            print('Help: cannot connect to server. '\
+            logger.error(e)
+            logger.error('Help: cannot connect to server. '\
                   'Check if server is dead or still spinning up.')
```

Comment:
You can consolidate this into a single call:
```suggestion
            logger.exception('Help: cannot connect to server. '\
                  'Check if server is dead or still spinning up.')
```